pkgs/development/python-modules/sfepy/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchurl
+, numpy
+, scipy
+, matplotlib
+, pyparsing
+, tables
+, cython
+, python
+, sympy
+}:
+
+buildPythonPackage rec {
+  name = "sfepy_${version}";
+  version = "2019.2";
+
+  src = fetchurl {
+    url="https://github.com/sfepy/sfepy/archive/release_${version}.tar.gz";
+    sha256 = "17dj0wbchcfa6x27yx4d4jix4z4nk6r2640xkqcsw0mf62x5l1pj";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    cython
+    scipy
+    matplotlib
+    pyparsing
+    tables
+    sympy
+  ];
+
+  checkPhase = ''
+    ${python.interpreter} setup.py build_ext --inplace
```

Comment:
This ... is unusual. `buildPythonPackage`'s `checkPhase` is run after install, so the installed package should already be available in the python environment, and the tests should be testing that installed state.

Granted, there's some weird stuff going on here anyway because `sfepy` seem to have written their own test framework for this.

Reply:
Thanks for looking at this. I was mainly copying the [travis.yml](https://github.com/sfepy/sfepy/blob/master/.travis.yml#L59), but I think you have a better approach below.